docs/package_templates/autotools_package/all/test_package/conanfile.py

```diff
@@ -4,11 +4,9 @@
 import os
 
 
-# It will become the standard on Conan 2.x
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
```

Comment:
```suggestion
    generators = "CMakeDeps", "CMakeToolchain"
```

Can probably drop the `VirtualBuildEnv` and `VirtualRunEnv` in most places as well.